ethsigner/core/src/main/java/tech/pegasys/ethsigner/core/EthSigner.java

```diff
@@ -138,11 +139,17 @@ private HttpServerOptions applyConfigTlsSettingsTo(final HttpServerOptions input
       final String password = readSecretFromFile(tlsConfig.getKeyStorePasswordFile().toPath());
       result.setPfxKeyCertOptions(new PfxOptions().setPath(keyStorePathname).setPassword(password));
 
-      if (tlsConfig.getKnownClientsFile().isPresent()) {
+      if (tlsConfig.getClientAuthConstraints().isPresent()) {
+        final ClientAuthConstraints constraints = tlsConfig.getClientAuthConstraints().get();
```

Comment:
be nice to move this block in the if into another small function to apply the tls trust options